### PR TITLE
Update all pods to latest

### PR DESCRIPTION
**Pull request by TheZoker**

### CODE REVIEW

**Comment by kordianbruck** [comment]
Shouldn't the pods be removed? https://github.com/TCA-Team/iOS/issues/20

File: TUM Campus App/CalendarViewController.swift [hunk 1]
```diff
@@ -121,7 +121,7 @@ extension CalendarViewController {
         if !sameDay(newDate, b: date) {
             newDate = date
         }
-        dayPlannerView.scroll(to: newDate, options: MGCDayPlannerScrollDateTime, animated: animated)
+        //dayPlannerView.scroll(to: newDate, options: MGCDayPlannerScrollDateTime, animated: animated)
```
Comment (TheZoker) [inline TUM Campus App/CalendarViewController.swift]: @mathiasquintero Can you have a look at this? This was the only issue I was not able to resolve, even after and hour of research. I just get this error:

> Use of unresolved identifier 'MGCDayPlannerScrollDateTime'

**Comment by TheZoker** [comment]
Nevertheless, the dependencies are updated in the Pod file and some little compatibility fixes are included.
Also XCode made some changes according to iOS 10.2 and XCode 8.2

File: Podfile [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 	pod 'TKSubmitTransition', :git => 'https://github.com/jvitor/TKSubmitTransition.git'
 end
 
 post_install do |installer|
-  installer.pods_project.targets.each do |target|
-    target.build_configurations.each do |config|
-      config.build_settings['SWIFT_VERSION'] = '3.0.1'
+    installer.pods_project.targets.each do |target|
+        target.build_configurations.each do |config|
+            config.build_settings['SWIFT_VERSION'] = '3.0'
```
Comment (mathiasquintero) [inline Podfile]: Are you sure about making it 3.0? I think some of the dependencies are actually written in 3.0.1

File: TUM Campus App/CalendarViewController.swift (hunk 1 above)
Comment (mathiasquintero) [inline TUM Campus App/CalendarViewController.swift]: That's a problem with the enum case. Try '.dateTime'

File: TUM Campus App/TUFilmCardCell.swift [hunk 3]
```diff
@@ -14,8 +14,8 @@ class TUFilmCardCell: CardTableViewCell {
     override func setElement(_ element: DataElement) {
         if let unwrappedMovie = element as? Movie{
             titleLabel.text = unwrappedMovie.text
-            let cal = Calendar.current
-            let components = (cal as NSCalendar).components([NSCalendar.Unit.month, NSCalendar.Unit.day], from: unwrappedMovie.airDate as Date)
+            let calendar = Calendar.current
+            let components = calendar.dateComponents([.month, .day], from: unwrappedMovie.airDate as Date)
```
Comment (mathiasquintero) [inline TUM Campus App/TUFilmCardCell.swift]: is the as Date necessary? This seems like some leftover useless cast from the swift3 update. It airDate still an NSDate? If so can you please change it?

File: Podfile (hunk 2 above)
Comment (TheZoker) [inline Podfile]: No, I'm not quite sure, why this changed. I reverted it (or actually update it to 3.0.2) now

File: TUM Campus App/CalendarViewController.swift (hunk 1 above)
Comment (TheZoker) [inline TUM Campus App/CalendarViewController.swift]: Works, thank you :)

File: TUM Campus App/TUFilmCardCell.swift (hunk 3 above)
Comment (TheZoker) [inline TUM Campus App/TUFilmCardCell.swift]: I'm not yet sure, if it's really necessary. It's just the update to Swift 3: http://stackoverflow.com/a/38933800/2977288

Comment (TheZoker) [inline TUM Campus App/TUFilmCardCell.swift]: I rewrote it, now it should be Swift 3 conform

**Comment by mathiasquintero** [comment]
Looking good. @mammuth please merge this into your fork if you don't want any conflicts

**Pull request merged**